bot.js

```diff
@@ -30,10 +32,22 @@ rtm.on(RTM_EVENTS.MESSAGE, function handleRtmMessage(message){
     console.log('Message: ', message);
     var channelId = message.channel;
 
-    if(message.text && message.text.indexOf(botIdentifier) > - 1 && message.user != botId){
-        if(message.text.indexOf('twitter') > -1){
+    if(message.text && message.text.indexOf(atBot) > - 1 && message.user != botId){
+        var user = message.text.replace(/ /g, '').replace(atBot, '').replace(':','');
+        var start = user.indexOf('<@');
+        var end = user.indexOf('>');
+        var userId = user.substring(start, end+1);
+        if(ids[userId] != null){
+            user = user.replace(userId, ids[userId])
+        } else {
+          var croppedid = userId.substring(2, userId.length-1)
+          var name =  rtm.dataStore.getUserById(croppedid).name
```

Comment:
when passing in an name directly (Ie cudabear instead of @cudabear) this line fails because rtm.datastore.getUserByid(croppedId) returns undefined